tutorials/cloud-run-eventing-kafka/kafka-cr-eventing/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Please remove this license file.

See this license at the top level of this repository, which applies the Apache 2.0 license to all code samples in this repository:
https://github.com/GoogleCloudPlatform/community/blob/master/LICENSE.md